api/src/main/java/org/openmrs/util/OpenmrsUtil.java

```diff
@@ -418,63 +418,7 @@ public static boolean folderContains(File folder, String filename) {
 	 * @param p properties from runtime configuration
 	 */
 	public static void startup(Properties p) {
-		
-		// Override global OpenMRS constants if specified by the user
```

Comment:
why did you remove the code here? it is still needed. You should not remove code so you can make a variable a constant. If you make a variable a constant and you get a compile error it just means that we are not able to make it a constant. The task of this ticket is simply to make things constants which are never reassigned.

Reply:
@teleivo I will do the necessary changes. Thanks